packages/core/src/components/cv-dropdown/cv-dropdown.vue

```diff
@@ -88,7 +88,10 @@
           :aria-labelledby="`${uid}-label`"
           ref="droplist"
         >
-          <slot></slot>
+          <slot v-if="!this.$props.items"></slot>
```

Comment:
Typically we have used the form where the slot overrides the prop content. 
```
<slot name="items">{{items}}</slot>
```

Do not access this.$props.items, use items directly e.g. `v-for="item in items"`

Reply:
Will resubmit pull request with change